editor/src/clj/editor/workspace.clj

```diff
@@ -467,17 +467,20 @@ ordinary paths."
   (or (:template resource-type)
       (some->> resource-type :ext (str "templates/template."))))
 
+(def ^:private user-resource-path "/templates/template.")
```

Comment:
It's not really the same path as path to a resource in the jar file because of `/` in the beginning. 
I think it's easier to use two separate variables to avoid mess here.

Reply:
it makes sense

Reply:
Fixed!